README.rst

```diff
@@ -28,114 +29,79 @@ README for Pylint - https://pylint.pycqa.org/
    :width: 200
    :alt: Tidelift
 
-.. list-table::
-   :widths: 10 100
+What is pylint ?
+================
 
-   * - |tideliftlogo|
-     - Professional support for pylint is available as part of the `Tidelift
-       Subscription`_.  Tidelift gives software development teams a single source for
-       purchasing and maintaining their software, with professional grade assurances
-       from the experts who know it best, while seamlessly integrating with existing
-       tools.
+.. Do not modify this without also modifying doc/index.rst
 
-.. _Tidelift Subscription: https://tidelift.com/subscription/pkg/pypi-pylint?utm_source=pypi-pylint&utm_medium=referral&utm_campaign=readme
+Pylint is a `static code analyser`_ for Python 2 or 3. The latest version supports Python
+3.7.2 and above.
 
+.. _`static code analyser`: https://en.wikipedia.org/wiki/Static_code_analysis
 
-======
-Pylint
-======
+Pylint analyses your code without actually running it. It checks for errors, enforces a coding
+standard, looks for `code smells`_, and can make suggestions about how the code could be refactored.
 
-**It's not just a linter that annoys you!**
+.. _`code smells`: https://martinfowler.com/bliki/CodeSmell.html
 
-Pylint is a Python static code analysis tool which looks for programming errors,
-helps enforcing a coding standard, sniffs for code smells and offers simple refactoring
-suggestions.
+Pylint can infer actual values from your code using its internal code representation (astroid).
+If your code is ``import logging as argparse``, Pylint will know that ``argparse.error(...)``
+is in fact a logging call and not an argparse call.
 
-It's highly configurable, having special pragmas to control its errors and warnings
-from within your code, as well as from an extensive configuration file.
-It is also possible to write your own plugins for adding your own checks or for
-extending pylint in one way or another.
```

Comment:
Probably need to include that in the intro for the documentation.

Reply:
Yeah I think that the configurability is something we should stress.

Reply:
Done.